### PR TITLE
document elasticsearch sink

### LINKED ISSUE #1292: document elasticsearch sink
### Related code PR

https://github.com/risingwavelabs/risingwave/pull/10357
https://github.com/risingwavelabs/risingwave/pull/10415

### Which part(s) of the docs might be affected or should be updated? And how?

here is the draft: 

https://www.notion.so/risingwave-labs/Sinking-Data-to-Elasticsearch-from-RisingWave-Draft-879f4f51acf9433ebc5e92891896b8ac?pvs=4

### Reference

_No response_

### LINKED ISSUE #1303: Document: feat(sink): support elasticsearch 8 sink
This issue tracks the documentation update needed for the merged PR #12269.

Source PR URL: https://github.com/risingwavelabs/risingwave/pull/12269
Source PR Merged At: 2023-09-14T06:34:27Z

### PR DESCRIPTION
**Pull request by hengm3467**
## Info

- **Description**

Document the support for sinking data to Elasticsearch (versions 7.x and 8.x)

- **Notes**

  [ Include any supplementary context or references here. ]

- **Related code PR**

  [ Provide a link to the relevant code PR here, if applicable. ]

- **Related doc issue**
  
  Resolves #1292 , #1303

## For reviewers

- **Preview**

  [ Paste the preview link to the updated page(s) here. Edit this item after the preview site is ready. To find the updated pages, scroll down to locate and open the Amplify preview link and select the **upcoming** version of the documentation. ]

- **Key points**

  [ Parts that may need revision or extra consideration. ]

## Before merging

- [ ] I have checked the doc site preview, and the updated parts look good.

- [ ] I have acquired the approval from the owner (and optionally the reviewers) of the code PR and at least one tech writer (`CharlieSYH`, `emile-00`, & `hengm3467`).

### CODE REVIEW

**Comment by hengm3467** [comment]
@wogouzi Please confirm if `force_append_only` and `primary_key` can/need to be specified for this sink connector.

**Comment by wugouzi** [comment]
There is one more option `delimiter` that customizes the delimiter. There is no `primary_key` and `force_append_only` options.

File: docs/data-delivery.md [hunk 1]
```diff
@@ -30,6 +30,10 @@ Currently, RisingWave supports the following sink connectors:
 
   With this connector, you can sink data from RisingWave to Delta Lake. For details about the syntax and parameters, see [Sink data to Delta Lake](/guides/sink-to-delta-lake.md).
 
+- Delta Lake sink connector (`connector = 'elasticsearch'`)
```
Comment (neverchanje) [inline docs/data-delivery.md]:
```suggestion
- Elasticsearch sink connector (`connector = 'elasticsearch'`)
```

**Review by neverchanje — APPROVED** [review]
https://docs.risingwave.com/docs/current/rw-integration-summary/#other-databases Thanks Heng. Please also include the support in the integration summary.

**Review by emile-00 — COMMENTED** [review]
Would be helpful to include an example but everything else looks good!

**Pull request merged**